manifests/exec_restorecon.pp

```diff
@@ -15,15 +16,25 @@
   Stdlib::Absolutepath $path        = $title,
   Boolean              $refreshonly = true,
   Boolean              $recurse     = true,
+  Boolean              $force       = false,
   Optional[String]     $unless      = undef,
   Optional[String]     $onlyif      = undef,
 ) {
+
   include ::selinux
-  $command = $recurse ? {
-    true  => 'restorecon -R',
-    false => 'restorecon',
+
+  $opt_recurse = $recurse ? {
+    true  => '-R',
+    false => '',
   }
 
+  $opt_force = $force ? {
+    true  => '-F',
+    false => '',
+  }
+
+  $command = "restorecon $opt_force $opt_recurse"
```

Comment:
enclosed vars need to be enclosed in {} (e.g. `${opt_force}`)

https://travis-ci.org/voxpupuli/puppet-selinux/jobs/257908521#L1051